devtools/src/main/resources/reference.conf

```diff
@@ -0,0 +1,35 @@
+kalix.dev-mode {
+  enabled = true
+
+  # settings to configure a Kalix Proxy container in dev-mode
+  proxy-container {
+
+    # Allows for overriding the Kalix Proxy image.
+    # When empty, the image use when the SDK was built will be selected.
+    proxy-image = ""
+    proxy-image = ${?KALIX_PROXY_IMAGE}
+
+    # Allows for overriding the port the Kalix Proxy will run.
+    # Default to 9000. Useful when running more than one service in dev-mode.
+    proxy-port = 9000
+
+    # Defines a unique identification name for this service.
+    # Useful when running more than one service and testing intra-service calls.
+    service-name = ""
+
+    # Enable ACL checks in development.
+    # ACL checks are disabled by default during development.
+    acl-enabled = false
+
+    # Enable advanced view features (multi-table joins).
+    view-features-all = false
+
+    # When configuring with a Kafka broker, this settings should point
+    # to a Kafka properties file, eg: /conf/kafka.properties.
+    broker-config-file = ""
+
+    # When running with a PubSub emulator, this settings must be configured
+    # to its host, eg: gcloud-pubsub-emulator.
+    pubsub-emulator-host = ""
```

Comment:
Didn't really try those two. Not yet working. 

A few more things will be needed, like mounting volume to read config (Kafka) and passing pubsub config to container.